src/common/utils.py

```diff
@@ -263,5 +263,6 @@ def read_json(file_path, default=None):
             log.error("Unable to read file from '%s'. Failed with exception: %s", file_path, e)
             raise
         else:
-            log.info("Unable to read file '%s' due to an exception: %s. Using default: %s", file_path, e, default)
+            if not isinstance(e, FileNotFoundError):
```

Comment:
This will limit the logs available to detect scenarios where the file can't be found but a default value is provided.
Is there a reason for avoiding logging of `FileNotFoundError` errors?

Reply:
I did by purpose, see PR description.
Now we have two files and it will become a common case to not have the run-instances-override.json file or anyway miss one of the two files and this `due to an exception: [Errno 2] No such file or directory` is a confusing message. 

I don't like printing the word "exception" and "Errno" if the file is missing, but there are no real errors, the file simply doesn't exist and this is the default and the most common case. 
Let's discuss about this change.

Reply:
Additional details:
* for files without a default we're still printing the error and failing.
* for files with a default (for which we were printing just an info message) we're now differentiating two cases:
  * if the file doesn't exist --> this is not a real error and this will be the default for most of the users --> no info message
  * if the file exists and there are parsing error --> show the info message but continue to use the default as before.

Reply:
Thanks for the explanation. Makes sense.